tests/scripts/all.sh

```diff
@@ -866,19 +866,18 @@ component_check_test_cases () {
     fi
     tests/scripts/check_test_cases.py $opt
     unset opt
+
+    # Check if some test are explicitely disabled when USE_PSA_CRYPTO is set
```

Comment:
Please add an `msg` line here — these checks belongs under “check test cases” but not under “check test case descriptions”.

Also, as a comment because it would be too verbose for `msg`, please explain why we're doing this — not because doing otherwise would break something, but as a matter of policy that we want to ensure there is no missed testing when `USE_PSA_CRYPTO` is enabled.